nps_active_space/utils/models.py

```diff
@@ -208,7 +438,8 @@ def __init__(self, data: gpd.GeoDataFrame, id_col: str, datetime_col: str, z_col
         if z_col:
             col_renames[z_col] = 'z'
         data.rename(columns=col_renames, inplace=True)
-        data.rename_geometry('geometry', inplace=True)
+        if 'geometry' not in data:
```

Comment:
Were you running into errors when the geometry column was not explicitly named "geometry"?

Reply:
yes